### PR TITLE
Features to Capstone Project 1 - Live Album Launch

### PR DESCRIPTION
**Pull request by Draxeytina**
-Add and update HTML files
-Add and update CSS files (styling)
-Add and update JavaScript files (functionality, dynamic HTML)
-Update README file

Kindly follow the link to view more about the project:
https://www.loom.com/share/be78e256066041ae9bfb00b521957101

### CODE REVIEW

File: home.css [hunk 1]
```diff
@@ -0,0 +1,811 @@
...
+    display: -webkit-box;
+    display: flex;
+    -webkit-box-orient: vertical;
+    -webkit-box-direction: normal;
+    -ms-flex-direction: column;
+    flex-direction: column;
+    -webkit-box-align: center;
+    -ms-flex-align: center;
+    align-items: center;
+    width: 90%;
```
Comment (omar-muhamad) [inline home.css]: - [x] Please make your header take the whole page width not boxed like you did (see image below) to match the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| image  | image  |

**[Hint]: You can do it just by changing the value of this width to be 100%**

File: about.css [hunk 2]
```diff
@@ -0,0 +1,953 @@
...
+    display: -webkit-box;
+    display: flex;
+    -webkit-box-orient: vertical;
+    -webkit-box-direction: normal;
+    -ms-flex-direction: column;
+    flex-direction: column;
+    -webkit-box-align: center;
+    -ms-flex-align: center;
+    align-items: center;
+    width: 90%;
```
Comment (omar-muhamad) [inline about.css]: - [x] Same here on the about page kindly make the header take the whole page width.

File: home.css [hunk 3]
```diff
@@ -0,0 +1,811 @@
...
+}
+
+#salutation {
+  color: #ec5242;
+  font-family: lato, sans-serif;
+  margin: 30px 0 5px 0;
+  font-size: 1.2rem;
+}
+
+#header-section h1 {
```
Comment (omar-muhamad) [inline home.css:45]: - [x] For your `header-section` div please fix all the elements in it adding the correct font size and font-weight for each element to match the given design.
- [x] Also please add the value of the correct margins between all those elements too and try to stick as much as possible to the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| image  | image |

File: home.css [hunk 4]
```diff
@@ -0,0 +1,811 @@
...
+  padding: 30px 15px;
+}
+
+#program .division-line {
+  width: 50px;
+  height: 2px;
+  background-color: #ec5242;
+  opacity: 0.6;
+  margin: 15px 0 30px 0;
+}
```
Comment (omar-muhamad) [inline home.css]: - [x] For the program section please use the correct font family to match the given design.
[Hint] you can choose any similar font to the given design from google fonts like `lato` or `roberto` fonts.

| Given Design  | Your Design |
| ------------- | ------------- |
| image  | image  |

File: home.css [hunk 5]
```diff
@@ -0,0 +1,811 @@
...
+
+#featured-h2 {
+  display: flex;
+  justify-content: center;
+  padding: 30px 15px;
+  padding-bottom: 0;
+  margin-bottom: 0;
+}
+
```
Comment (omar-muhamad) [inline home.css:192]: - [x] The featured artists section looks awesome, Good job 💪

File: about.css [hunk 6]
```diff
@@ -0,0 +1,953 @@
...
+  font-family: cocogoose, sans-serif;
+  margin-left: 3px;
+}
+
+#footer-p {
+  font-size: 1rem;
+  width: 35%;
+  padding-top: 0;
+  margin-top: 0;
+}
```
Comment (omar-muhamad) [inline about.css:205]: - [x] The footer on the about page suppose to have a dark background like the given design. So please fix that.
- [x] please add the correct height to the footer on both the main page and about page and kindly center the elements inside it too, and try to stick as much as possible to the given Given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| image  | image  |

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @Draxeytina,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [x] Please check the inline comments below.

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: about.css (hunk 2 above)
Comment (Draxeytina) [reply about.css]: Thank you. Done.

File: home.css (hunk 1 above)
Comment (Draxeytina) [reply home.css]: Noted @Omar-Muhamad, I had overlooked it.

File: home.css (hunk 3 above)
Comment (Draxeytina) [reply home.css:45]: Thank you @Omar-Muhamad. I adjusted accordingly.

File: home.css (hunk 4 above)
Comment (Draxeytina) [reply home.css]: Thank you for the suggestions. I applied Lato 😄

File: about.css (hunk 6 above)
Comment (Draxeytina) [reply about.css:205]: Thank you. I adjusted the height, gave it a dark background, and centered the contents.

image

File: about.html [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+          <span>Logitap</span>
+          <span>Magic Pixels</span>
+          <span>Park Cafeteria</span>
+          <span>Mura Meats</span>
+          <span>Hillside Kitchen</span>
+          <span>Melz Cakes</span>
+        </div>
+      </section>
+    </main>
+    <footer>
```
Comment (Ismailco) [inline about.html:118]: - [x] [Optional] The footer should be the same for the ```Home page``` and the ```About page```, the only difference is the color, please reduce the padding and consider moving the logo to the left just like in the home page

- it should be something like this:
Screenshot_20220519_214632

File: index.html [hunk 8]
```diff
@@ -0,0 +1,164 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./home.css">
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/2e9eda9b73.js" crossorigin="anonymous"></script>
+    <title>Mellisa Live Album Launch</title>
+</head>
+  <body>
+    <header>
+    <nav id="nav">
```
Comment (Ismailco) [inline index.html:15]: - [x] [Optional] The top bar has too much padding, kindly reduce the padding on the top bar and remove the small margin at the top

|Your design|CC design|
|---|---|
|Screenshot_20220519_214810|Screenshot_20220519_215247|

**Review by Ismailco — CHANGES_REQUESTED** [review]
#### Hi @Draxeytina,

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

giphy

## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html (hunk 8 above)
Comment (Draxeytina) [reply index.html:15]: Yes true. I removed padding and margin-top. I also changed the color 😄 something I noticed after you mentioned. Thanks @Ismailco

File: about.html (hunk 7 above)
Comment (Draxeytina) [reply about.html:118]: Thank you, implemented the suggestion.

File: README.md [hunk 9]
```diff
@@ -1 +1,44 @@
...
+This is part of Microverse capstone project 1 to demonstrate understanding of HTML5, CSS3, JavaScript 
+and related frameworks.
 
+## Built With
+- HTML5
+- CSS3
+- JavaScript
+- Flexbox
+- CSS Grid
+
```
Comment (ShahierNashaat) [inline README.md:11]: - [x] [Optional] Please leave a link to the recorded video presented in the README file.

**Review by ShahierNashaat — APPROVED** [review]
Hi @Draxeytina , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 9 above)
Comment (Draxeytina) [reply README.md:11]: Thank you so much @ShahierNashaat! I think it's important to place the link there. I'll do so.

**Pull request merged**